aiidalab_widgets_base/aiida_viewers.py

```diff
@@ -74,7 +76,28 @@ def __init__(self, structure, downloadable=True, **kwargs):
             self.download_btn = ipw.Button(description="Download")
             self.download_btn.on_click(self.download)
             children.append(ipw.HBox([self.file_format, self.download_btn]))
-        super(StructureDataVisualizer, self).__init__(children, **kwargs)
+        super().__init__(children, **kwargs)
+
+    def update_structure(self, structure):
+        """Update structure.
+        :param structure: Structure to be visualized
+        :type structure: StructureData, CifData, Atoms (ASE)"""
+        from ase import Atoms
+        self._structure = structure if isinstance(structure, Atoms) else structure.get_ase()
+        self.refresh_view()
+
+    def reset_view(self):
```

Comment:
There is no much logic here, just a replacement for the built-in `clear()` function. But since the function is working properly now - I deleted this function.

Reply:
But I agree that it does not make much sense to keep it as a separate function.